maven2-plugins/myfaces-builder-plugin/src/main/java/org/apache/myfaces/buildtools/maven2/plugin/builder/MakeComponentsMojo.java

```diff
@@ -360,7 +380,10 @@ else if (!outFile.exists())
         for (Iterator i = sourceDirs.iterator(); i.hasNext();)
         {
             String srcDir = (String) i.next();
-            builder.addSourceTree(new File(srcDir));
+            File dir = new File(srcDir);
+            //recusively add files to the builder
```

Comment:
Maybe just clean up the formatting of the comment to be consistent but I wouldn't block PR approval on that!